components/mediation/mediators/oauth-mediator/org.wso2.micro.integrator.mediator.oauth/src/main/java/org/wso2/micro/integrator/mediator/oauth/OAuth2TokenValidationServiceClient.java

```diff
@@ -0,0 +1,68 @@
+package org.wso2.micro.integrator.mediator.oauth;
+
+import java.rmi.RemoteException;
+import java.util.List;
+
+import org.apache.axis2.AxisFault;
+import org.apache.axis2.context.ConfigurationContext;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.oauth2.stub.OAuth2TokenValidationServiceStub;
+import org.wso2.carbon.identity.oauth2.stub.dto.OAuth2TokenValidationRequestDTO;
+import org.wso2.carbon.identity.oauth2.stub.dto.OAuth2TokenValidationRequestDTO_OAuth2AccessToken;
+import org.wso2.carbon.identity.oauth2.stub.dto.OAuth2TokenValidationRequestDTO_TokenValidationContextParam;
+import org.wso2.carbon.identity.oauth2.stub.dto.OAuth2TokenValidationResponseDTO;
+import org.wso2.micro.core.util.MicroCarbonUtils;
+
+public class OAuth2TokenValidationServiceClient {
+
+    private OAuth2TokenValidationServiceStub stub = null;
+    private static final Log log = LogFactory.getLog(OAuth2TokenValidationServiceClient.class);
+
+    /**
+     * OAuth2TokenValidationService Admin Service Client
+     *
+     * @param backendServerURL
```

Comment:
Shall we add descriptions for the input paramters?

Reply:
Done in 563f10f6e2d2e0936ec3843f9ae35d84a369bef9